### PR TITLE
Added IQ audio documentation

### PR DESCRIPTION
**Pull request by aallan**
Pulled in from https://datasheets.raspberrypi.com/iqaudio/iqaudio-product-brief.pdf this is a stub for documentation around the IQ audio boards. This has a bunch of problems ranging from stylistic issues through structural ones. On the other hand it's somewhere to start from, and I'm pondering whether I should just merge this as is rather than keep it on a branch.  This PR to serve as a placeholder so I remember I'm going to want to merge it at some point.

### CODE REVIEW

File: documentation/asciidoc/accessories/iqaudio/configuration.adoc [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+
+Each input and output device has its own “mixer” allowing the audio levels / volume to be adjusted
+independently. Within the codec itself other mixers / switches exist to allow the output to be Mono’d
+for single speaker output, signals may also be inverted and there is a 5 band Equaliser to adjust
+certain frequency bands. These settings can be controlled through alsamixer interactively or
+programatically.
+
+It is important to note that the AUX IN and AUX OUT are both 1vRMS. It may be necessary to adjust
+the AUX IN’s mixer to ensure the input signal doesn’t saturate the ADCs. (Analogue to Digital
+Convertors). Similarly, the output mixers may been to be adjusted to get the best possible output.
```
Comment (RuffaloLavoisier) [inline documentation/asciidoc/accessories/iqaudio/configuration.adoc:84]: `may been` -> `may be` ?

File: documentation/asciidoc/accessories/iqaudio/configuration.adoc [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+The IQaudio Codec Zero board uses the Dialog Semiconductor DA7212 codec. The DA7212 allows
+the recording of audio from the board’s built in MEMS microphone, from stereo Phono sockets (AUX
+IN), 2x mono external Electret microphones and playback through stereo Phono sockets (AUX OUT)
+and mono speaker connector.
+
+Each input and output device has its own “mixer” allowing the audio levels / volume to be adjusted
+independently. Within the codec itself other mixers / switches exist to allow the output to be Mono’d
+for single speaker output, signals may also be inverted and there is a 5 band Equaliser to adjust
+certain frequency bands. These settings can be controlled through alsamixer interactively or
+programatically.
```
Comment (RuffaloLavoisier) [inline documentation/asciidoc/accessories/iqaudio/configuration.adoc:80]: programatically -> program**m**atically

File: documentation/asciidoc/accessories/iqaudio/configuration.adoc [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+Raspberry Pi. No screen needs to be attached to the Pi itself.
+
+Other “headless” applications install a web server onto the Pi, so to control it you simply open a
+browser window on your PC / Mac / Tablet or similar and interact with the Pi based application
+through a web page. 
+
+NOTE: Applications such as https://www.max2play.com/en/[Max2Play], https://moodeaudio.org/[MoodeAudio], https://volumio.com/en/[Volumio] and https://www.runeaudio.com/[RuneAudio] work.
+this way.
+
+NOTE: Most audio applications assume they are the only thing running on the Pi so most require a dedicated SD card. The
```
Comment (RuffaloLavoisier) [inline documentation/asciidoc/accessories/iqaudio/configuration.adoc:49]: on the Pi so most -> on the Raspberry Pi so most

File: documentation/asciidoc/accessories/iqaudio/configuration.adoc [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+turns the Raspberry Pi into a Roon Certified end-point. The Pi is then controlled form the RoonLabs
+PC/Mac application which connects to and plays music through the IQaudio sound card on your
+Raspberry Pi. No screen needs to be attached to the Pi itself.
+
+Other “headless” applications install a web server onto the Pi, so to control it you simply open a
+browser window on your PC / Mac / Tablet or similar and interact with the Pi based application
+through a web page. 
+
+NOTE: Applications such as https://www.max2play.com/en/[Max2Play], https://moodeaudio.org/[MoodeAudio], https://volumio.com/en/[Volumio] and https://www.runeaudio.com/[RuneAudio] work.
+this way.
```
Comment (RuffaloLavoisier) [inline documentation/asciidoc/accessories/iqaudio/configuration.adoc:47]: What is that? Is there no additional explanation?

File: documentation/asciidoc/accessories/iqaudio/configuration.adoc [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+So if there’s no screen how do you control it?
+
+Well in some cases you don’t need to do anything with the Pi at all. Take the https://roonlabs.com/[RoonLabs] images, that
+turns the Raspberry Pi into a Roon Certified end-point. The Pi is then controlled form the RoonLabs
+PC/Mac application which connects to and plays music through the IQaudio sound card on your
+Raspberry Pi. No screen needs to be attached to the Pi itself.
+
+Other “headless” applications install a web server onto the Pi, so to control it you simply open a
+browser window on your PC / Mac / Tablet or similar and interact with the Pi based application
+through a web page.
```
Comment (RuffaloLavoisier) [inline documentation/asciidoc/accessories/iqaudio/configuration.adoc:44]: Why don't you get rid of the line change?

File: documentation/asciidoc/accessories/iqaudio/configuration.adoc [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+=== Headless software
+
+Many of the software packages used with the Raspberry Pi simply turn the Pi into a “Headless” endpoint. That is to say that it doesn't need, or expect, a screen/display to be physically attached to the Raspberry Pi’s HDMI socket
+
+So if there’s no screen how do you control it?
+
+Well in some cases you don’t need to do anything with the Pi at all. Take the https://roonlabs.com/[RoonLabs] images, that
+turns the Raspberry Pi into a Roon Certified end-point. The Pi is then controlled form the RoonLabs
+PC/Mac application which connects to and plays music through the IQaudio sound card on your
+Raspberry Pi. No screen needs to be attached to the Pi itself.
```
Comment (RuffaloLavoisier) [inline documentation/asciidoc/accessories/iqaudio/configuration.adoc:40]: Why don't you get rid of the line change? and

to the Pi itself. -> to the Raspberry Pi itself.

File: documentation/asciidoc/accessories/iqaudio/configuration.adoc [hunk 7]
```diff
@@ -0,0 +1,134 @@
...
+#dtparam=audio=on
+----
+
+=== Headless software
+
+Many of the software packages used with the Raspberry Pi simply turn the Pi into a “Headless” endpoint. That is to say that it doesn't need, or expect, a screen/display to be physically attached to the Raspberry Pi’s HDMI socket
+
+So if there’s no screen how do you control it?
+
+Well in some cases you don’t need to do anything with the Pi at all. Take the https://roonlabs.com/[RoonLabs] images, that
```
Comment (RuffaloLavoisier) [inline documentation/asciidoc/accessories/iqaudio/configuration.adoc:37]: Raspberry Pi

File: documentation/asciidoc/accessories/iqaudio/configuration.adoc [hunk 8]
```diff
@@ -0,0 +1,134 @@
...
+turns the Raspberry Pi into a Roon Certified end-point. The Pi is then controlled form the RoonLabs
```
Comment (RuffaloLavoisier) [inline documentation/asciidoc/accessories/iqaudio/configuration.adoc:38]: Same, Raspberry Pi

File: documentation/asciidoc/accessories/iqaudio/configuration.adoc [hunk 9]
```diff
@@ -0,0 +1,134 @@
+== Configuration
+
+All our latest full size audio boards include a pre-programmed EEPROM. This allows Raspberry Pi
+OS to auto detect and auto configure itself allowing the IQaudio boards to be plug and play.
+
+If you need to configure Raspberry Pi OS yourself, you can edit the boot configuration file (found at
+`/boot/config.txt`) to manually add the appropriate device-tree overlay:
```
Comment (RuffaloLavoisier) [inline documentation/asciidoc/accessories/iqaudio/configuration.adoc:7]: Why don't you get rid of the line change?

File: documentation/asciidoc/accessories/iqaudio/configuration.adoc [hunk 10]
```diff
@@ -0,0 +1,134 @@
+== Configuration
+
+All our latest full size audio boards include a pre-programmed EEPROM. This allows Raspberry Pi
+OS to auto detect and auto configure itself allowing the IQaudio boards to be plug and play.
```
Comment (RuffaloLavoisier) [inline documentation/asciidoc/accessories/iqaudio/configuration.adoc:4]: Why don't you get rid of the line change?

File: documentation/asciidoc/accessories/iqaudio/advanced.adoc [hunk 11]
```diff
@@ -0,0 +1,40 @@
...
+* GPIO 23/24 for Rotary Encoder (physical volume control) or status LED (Codec Zero)
+* GPIO 25 for the IR Sensor.
+* GPIO 27 for the Rotary Encoder push switch / Code Zero switch
+
+=== DAC PRO, DAC{plus}, DigiAMP{plus}, Codec Zero
+
+image::images/pin_table.png[width="80%"]
+
+The DAC PRO, DAC{plus}, DigiAMP{plus} re-expose the Raspberry Pi signals allowing additional sensors etc
+to be added easily. Please note that some signals are for exclusive use (I2S and EEPROM) for some
```
Comment (RuffaloLavoisier) [inline documentation/asciidoc/accessories/iqaudio/advanced.adoc:28]: remove a `\n` or do you need a `.`?

File: documentation/asciidoc/accessories/iqaudio/digiamp_plus.adoc [hunk 12]
```diff
@@ -0,0 +1,31 @@
...
+* Uses the Texas Instruments TAS5756M PowerDAC
+
+WARNING: DO NOT APPLY POWER TO THE RASPBERRY PI’s OWN POWER INPUT WHEN USING DIGIAMP{plus}.
+
+image::images/digiamp_plus.png[width="80%"]
+
+NOTE: DigiAMP{plus} is a HAT compliant add-on accessory for the Raspberry Pi.
+
+DigiAMP{plus} uses the Texas Instruments TAS5756M PowerDAC to deliver direct connection to
+PASSIVE stereo speakers at up to 2×35Watts per channel with variable output. Ideal for a Pi based HiFi.
```
Comment (RuffaloLavoisier) [inline documentation/asciidoc/accessories/iqaudio/digiamp_plus.adoc:15]: Raspberry Pi ?

**Review by RuffaloLavoisier — COMMENTED** [review]
In my humble opinion, I think we need to get rid of `\n` that breaks words.

**Comment by lurch** [comment]
This is very early-stage documentation, and will likely be seeing major changes before we publish it. So not really worth commenting on it yet @RuffaloLavoisier :wink:

File: documentation/asciidoc/accessories/iqaudio/introduction.adoc [hunk 13]
```diff
@@ -1,49 +1,32 @@
-== Overview of IQAudio products
+== Overview of Raspberry Pi Audio
+ 
+Introducing our range of boards designed to bring high quality audio to your existing Hi-Fi or Raspberry Pi based equipment and projects.
 
-IQaudio sound cards are all designed to work with the current Raspberry Pi and those that have a
-40way male GPIO pin header. This includes Raspberry Pi 4B, 3B{plus}, 3A{plus}, 3, Zero etc. 
+With 4 different boards to choose from all our audio boards will fit any Raspberry Pi utilising a 40-pin GPIO header.
```
Comment (lurch) [inline documentation/asciidoc/accessories/iqaudio/introduction.adoc:5]: This sentence _probably_ isn't necessary, as none of the other HAT pages at https://www.raspberrypi.com/documentation/accessories/ mention that a 40-pin GPIO header is required? Every Raspberry Pi since the B+ has had a 40-pin header, and that was [released](https://www.raspberrypi.com/products/raspberry-pi-1-model-b-plus/) way back in 2014 :wink:

File: documentation/asciidoc/accessories/iqaudio/introduction.adoc [hunk 14]
```diff
@@ -1,49 +1,32 @@
...
+Balanced Out:: An XLR connector - Normally a 3 pin male connector. This is used in studio and some
+high-end Hi-Fi systems. It can also be used to drive active “monitor” speakers as used at discos or on
+stage directed towards the DJ or performers. 
+Stereo Speakers:: 2 sets of screw terminals for 2 x 25W speakers. These are for normal Hi-Fi speakers such as bookshelf or floor
+standing. These are known as passive speakers.
+Mono Speaker:: Screw terminal for single 1.2W speaker, as found in a transistor radio or similar.
+Headphones:: 3.5mm jack socket delivering stereo audio for a set of normal headphones. Our headphone amplifiers on the DAC boards can drive up to 80/90ohm impedance headphones. The headphone output of the Codec+ is also aux out.
+Aux In:: A double Phono/RCA connector or 3.5mm socket. Accepts analogue audio in up to 1vRMS. This can be used to record audio from a variable analogue source such as a mobile phone, MP3 player or similar.
+Aux Out:: A double Phono/RCA connector or 3.5mm socket. Delivers analogue audio out up to 1vRMS. This can be used to feed audio into an amplifier at a reduced volume compared to Line Out.
```
Comment (lurch) [inline documentation/asciidoc/accessories/iqaudio/introduction.adoc:31]: Might be worth adding a bit more explanatory text about the difference between "Line Out" and "Aux Out"? :man_shrugging:

File: documentation/asciidoc/accessories/iqaudio/introduction.adoc [hunk 15]
```diff
@@ -1,49 +1,32 @@
...
-similar.
-Aux Out:: A double Phono/RCA connector or 3.5mm socket. Delivers analogue audio up to 1vRMS
-output. Can be used to feed audio into an amplifier (but at a reduced volume compared to Line Out)
-Mic In:: a 3.5mm socket for use with an external Electret microphone. Some of our boards have a
-built in MEMS microphone that is disabled when the external microphone jack is inserted into the
-3.5mm socket.
+Line Out:: A double Phono/RCA connector - Normally red and white in colour. This output is a variable
+analogue signal (0..2vRMS) and can connect to your existing Hi-Fi (pre-amp or amplifier) or can be used
+to drive active speakers (with their own amplifier built in).
+Balanced Out:: An XLR connector - Normally a 3 pin male connector. This is used in studio and some
```
Comment (lurch) [inline documentation/asciidoc/accessories/iqaudio/introduction.adoc:23]: Is "This is used in studio" missing an additional noun?

File: documentation/asciidoc/accessories/iqaudio/introduction.adoc [hunk 16]
```diff
@@ -1,49 +1,32 @@
...
+stage directed towards the DJ or performers. 
+Stereo Speakers:: 2 sets of screw terminals for 2 x 25W speakers. These are for normal Hi-Fi speakers such as bookshelf or floor
+standing. These are known as passive speakers.
+Mono Speaker:: Screw terminal for single 1.2W speaker, as found in a transistor radio or similar.
+Headphones:: 3.5mm jack socket delivering stereo audio for a set of normal headphones. Our headphone amplifiers on the DAC boards can drive up to 80/90ohm impedance headphones. The headphone output of the Codec+ is also aux out.
+Aux In:: A double Phono/RCA connector or 3.5mm socket. Accepts analogue audio in up to 1vRMS. This can be used to record audio from a variable analogue source such as a mobile phone, MP3 player or similar.
+Aux Out:: A double Phono/RCA connector or 3.5mm socket. Delivers analogue audio out up to 1vRMS. This can be used to feed audio into an amplifier at a reduced volume compared to Line Out.
+Mic In:: A 3.5mm socket for use with an external Electret microphone. As the Codec Zero has a built in MEMS microphone this is disabled when the external Mic In is used.
```
Comment (lurch) [inline documentation/asciidoc/accessories/iqaudio/introduction.adoc:32]: Personally, I think I'd reword "As the Codec Zero has a built in MEMS microphone this is disabled when the external Mic In is used." to something like "The built-in MEMS microphone on the Codec Zero will be disabled when the external Mic In connector is used."

File: documentation/asciidoc/accessories/iqaudio/introduction.adoc [hunk 17]
```diff
@@ -1,49 +1,32 @@
...
-Aux In:: A double Phono/RCA connector or 3.5mm socket. Accepts analogue audio up to 1vRMS
-input. Can be used to record audio from a variable analogue source such as phone / tape recorder or
-similar.
-Aux Out:: A double Phono/RCA connector or 3.5mm socket. Delivers analogue audio up to 1vRMS
-output. Can be used to feed audio into an amplifier (but at a reduced volume compared to Line Out)
-Mic In:: a 3.5mm socket for use with an external Electret microphone. Some of our boards have a
-built in MEMS microphone that is disabled when the external microphone jack is inserted into the
-3.5mm socket.
+Line Out:: A double Phono/RCA connector - Normally red and white in colour. This output is a variable
+analogue signal (0..2vRMS) and can connect to your existing Hi-Fi (pre-amp or amplifier) or can be used
```
Comment (lurch) [inline documentation/asciidoc/accessories/iqaudio/introduction.adoc:21]: Perhaps "(0..2vRMS)" should be changed to "(0-2V RMS)" to match https://www.raspberrypi.com/products/iqaudio-dac-plus/ ?

File: documentation/asciidoc/accessories/iqaudio/introduction.adoc (hunk 16 above)
Comment (lurch) [reply documentation/asciidoc/accessories/iqaudio/introduction.adoc:32]: ...although perhaps the "on the Codec Zero" part isn't even necessary, if that's the only board with an external Mic In connector?

File: documentation/asciidoc/accessories/iqaudio/codec_zero.adoc [hunk 18]
```diff
@@ -24,8 +24,8 @@ Built in MEMS microphone and supports mono Electret microphone or mono speaker (
 It also has GREEN (GPIO23) and RED (GPIO24) LEDs, that can be used for status and a tactile
 button (GPIO27) that can be used to start an event.
 
-The Codec Zero makes a great starting point for a walkie talkie, intelligent doorbell, vintage radio
-hack or Smart type speaker.
+The Codec Zero makes a great starting point for many projects such as a walkie talkie, intelligent doorbell, vintage radio
+hack or smart type speaker.
```
Comment (lurch) [inline documentation/asciidoc/accessories/iqaudio/codec_zero.adoc:28]: "smart type speaker" -> "smart speaker"?

File: documentation/asciidoc/accessories/iqaudio/codec_zero.adoc [hunk 19]
```diff
@@ -1,17 +1,17 @@
 === Raspberry Pi Codec Zero
 
-Raspberry Pi Codec Zero is a Raspberry Pi Zero sized audio HAT. It delivers bi-directional digital audio signals (I2S) between Raspberry Pi Zero and its onboard Dialog Semiconductor DA7212 codec. Codec Zero supports a range of input and output devices,
+Raspberry Pi Codec Zero is a Raspberry Pi Zero sized audio HAT (Hardware Attached on Top). It delivers bi-directional digital audio signals (I2S) between Raspberry Pi Zero and its onboard Dialog Semiconductor DA7212 codec. Codec Zero supports a range of input and output devices.
 
 * High performance 24-bit audio codec
 * Supports common audio sample rates between 8-96kHz
-* Built in MEMS microphone (Mic2)
+* Built in MEMS (microelectromechanical) microphone (Mic2)
```
Comment (lurch) [inline documentation/asciidoc/accessories/iqaudio/codec_zero.adoc:7]: Should probably capitalise this as MicroElectroMechanical to make the acronym more obvious? :shrug: (but that's probably a question for @helenlynn )
Although https://en.wikipedia.org/wiki/Microelectromechanical_systems suggests that "micro-electro-mechanical systems" is a valid alternative.

Comment (helenlynn) [reply documentation/asciidoc/accessories/iqaudio/codec_zero.adoc:7]: I'd say leave it as it is. Camel case and hyphens would be obtrusive clutter here, and the acronym is probably not too hard to spot without them for most people.

File: documentation/asciidoc/accessories/iqaudio/configuration.adoc [hunk 20]
```diff
@@ -1,11 +1,13 @@
 == Configuration
 
-All our latest full size audio boards include a pre-programmed EEPROM. This allows Raspberry Pi
-OS to auto detect and auto configure itself allowing the IQaudio boards to be plug and play.
+All audio boards include a pre-programmed EEPROM (Electrically Erasable Programmable Read-only Memory), a form of computer memory that does not lose its content when the power supply is cut off and that can be erased and reused. This allows Raspberry Pi OS to autodetect and autoconfigure itself allowing Raspberry Pi audio boards to be plug and play.
```
[the remark this answers is not in the document]
Comment (lurch) [reply documentation/asciidoc/accessories/iqaudio/configuration.adoc:3]: Also, _if_ this statement is true, does it mean that `config.txt` _doesn't_ need to be manually edited? :thinking:

File: documentation/asciidoc/accessories/iqaudio/configuration.adoc [hunk 21]
```diff
@@ -1,11 +1,13 @@
...
 
-If you need to configure Raspberry Pi OS yourself, you can edit the boot configuration file (found at
-`/boot/config.txt`) to manually add the appropriate device-tree overlay:
+If you need to configure Raspberry Pi OS yourself, you can edit the boot configuration file. Comments may be added, or existing config values may be commented out and disabled, by starting a line with the `#` character. Edit and save the file with with the line appropriate to your board using the following command:
```
Comment (lurch) [inline documentation/asciidoc/accessories/iqaudio/configuration.adoc:5]: Could probably just link to https://www.raspberrypi.com/documentation/computers/config_txt.html rather than repeating the "how to edit config.txt" instructions :slightly_smiling_face:

File: documentation/asciidoc/accessories/iqaudio/configuration.adoc [hunk 22]
```diff
@@ -15,12 +17,14 @@ If you need to configure Raspberry Pi OS yourself, you can edit the boot configu
 | Codec Zero | `dtoverlay=iqaudio-codec`
 |===
 
+Alternatively, the `/boot/config.txt` file can be edited directly onto the Raspberry Pi Boot SD card inserted into your usual computer.
+
 === Disabling on-board audio
 
-To make your IQaudio sound card the primary audio device in Raspberry Pi OS you will need to
+To make your Raspberry Pi sound card the primary audio device in Raspberry Pi OS you will need to
```
Comment (lurch) [inline documentation/asciidoc/accessories/iqaudio/configuration.adoc:24]: "sound card" or "audio board"? :stuck_out_tongue_winking_eye:

File: documentation/asciidoc/accessories/iqaudio/configuration.adoc [hunk 23]
```diff
@@ -15,12 +17,14 @@ If you need to configure Raspberry Pi OS yourself, you can edit the boot configu
...
 disable the Raspberry Pi’s on-board audio card. This is done simply by commenting out the
```
Comment (lurch) [inline documentation/asciidoc/accessories/iqaudio/configuration.adoc:25]: "on-board audio card" doesn't make sense, so should probably be "on-board audio device" or just "on-board audio" or "built-in audio" or "internal audio device" ?

File: documentation/asciidoc/accessories/iqaudio/configuration.adoc [hunk 24]
```diff
@@ -52,34 +56,33 @@ file will need to be written to the Pi’s SD card to allow the Pi to boot.
...
-general no soldering is required to the IQaudio boards for normal operation.
+The Raspberry Pi range of sound cards attach to the Raspberry Pi’s 2x20 way pin header (GPIO header).
+They are designed to be supported on the Raspberry Pi using the supplied circuit board standoffs and screws. In
+general no soldering is required on the Raspberry Pi audio boards for normal operation with the exception when utilising hardwired connections for specific connectors, XLR (External Line Return) connections on the DAC Pro for example.
 
-The IQaudio range of boards are normally supplied with the necessary mounting hardware (PCB
-spacers and screws), these are for our latest low profile boards 4x 9mm PCB spacers and 8x M2.5
-screws. If additional parts (such as 2piece speaker connectors) are required these are normally
-provided too.
+The Raspberry Pi range of boards are normally supplied with the necessary mounting hardware (PCB
```
Comment (lurch) [inline documentation/asciidoc/accessories/iqaudio/configuration.adoc:63]: Delete "normally"?

File: documentation/asciidoc/accessories/iqaudio/configuration.adoc [hunk 25]
```diff
@@ -52,34 +56,33 @@ file will need to be written to the Pi’s SD card to allow the Pi to boot.
...
+spacers and screws), these are for our latest low profile boards 4 x 9mm PCB spacers and 8 x M2.5mm
+screws. If additional parts (such as 2 piece speaker connectors) are required these are normally
```
Comment (lurch) [inline documentation/asciidoc/accessories/iqaudio/configuration.adoc:65]: "If additional parts" should be reworded, and "normally" should definitely be deleted.

File: documentation/asciidoc/accessories/iqaudio/configuration.adoc [hunk 26]
```diff
@@ -52,34 +56,33 @@ file will need to be written to the Pi’s SD card to allow the Pi to boot.
...
+included.
 
 image::images/hat.png[width="80%"]
 
-The PCB spacers should be screwed (finger tight only) to the Raspberry Pi before adding the IQaudio
+The PCB spacers should be screwed, finger tight only, to the Raspberry Pi before adding the audio
```
Comment (lurch) [inline documentation/asciidoc/accessories/iqaudio/configuration.adoc:70]: Should finger-tight be hyphenated?

File: documentation/asciidoc/accessories/iqaudio/configuration.adoc [hunk 27]
```diff
@@ -52,34 +56,33 @@ file will need to be written to the Pi’s SD card to allow the Pi to boot.
 
 === Attaching the HAT
 
-The IQaudio range of sound cards attach to the Raspberry Pi’s 2x20 way pin header (GPIO header).
-They are designed to be supported on the Pi using the supplied PCB standoffs and screws. In
-general no soldering is required to the IQaudio boards for normal operation.
+The Raspberry Pi range of sound cards attach to the Raspberry Pi’s 2x20 way pin header (GPIO header).
```
[the remark this answers is not in the document]
Comment (aallan) [reply documentation/asciidoc/accessories/iqaudio/configuration.adoc:59]: i think I normally write 40-pin header, mainly because there aren't 40 GPIO pins, so writing 40-pin GPIO header is technically incorrect. Which is the best type of incorrect.

[the remark this answers is not in the document]
Comment (aallan) [reply documentation/asciidoc/accessories/iqaudio/configuration.adoc:59]: We definitely don't refer to it anywhere as 2x20 pins.

**Comment by aallan** [comment]
With the latest diagrams added, I think we're ready for review.

**Comment by aallan** [comment]
Paging @LizUpton or @helenlynn for copy edit.

**Comment by aallan** [comment]
Waiting for #2639, #2641, #2642, #2643, #2644, #2647, #2648, #2649, #2650, and #2651 to merge.

**Comment by andrum99** [comment]
> Waiting for ... #2647 ... to merge.

Don't need to wait for that one - it's unrelated.

**Review by lurch — APPROVED** [review]
